backend/controller/cronjobs/cronjobs.go

```diff
@@ -260,20 +270,24 @@ func (s *Service) killOldJobs(ctx context.Context) (time.Duration, error) {
 		pattern, err := cron.Parse(stale.Schedule)
 		if err != nil {
 			logger.Errorf(err, "Could not kill stale cron job %q because schedule could not be parsed: %q", stale.Key, stale.Schedule)
+			observability.Cron.JobFailed(ctx, stale.Key, stale.DeploymentKey)
```

Comment:
Instead of bundling all the failures to kill a job with failures of the actual job itself, could we refactor this to the following?

- JobRun with attribute `.succeeded`
- JobKilled with attribute `.succeeded`

Reply:
I like the idea and would like to take it a little further. Removing the `jobFailures` counter does result in a loss of visibility that can be avoided if the `ftl.status.succeeded` attribute can take on a more generalized semantic. Changing from success/fail to "outcome" would help. Outcome is a string representing the canonical set of outcomes for a given domain. So in the metrics where the `ftl.status.succeed` is used today; outcome would have the values of `success` or `fail`. 

For cron jobs I would like outcomes covering the following: success, failed to start, execution failure, and killed. This would allow me to remove the `jobsKilled` metric and segment latency by outcome

Reply:
updated